src/simulations/simulator.py

```diff
@@ -158,24 +150,26 @@ def run_one(self):
     actual_ids = []
     for campaign_ids in set_generator:
       actual_ids.append(campaign_ids)
-      sketch = self.estimator_config.sketch_factory(sketch_random_seed)
+      sketch = self.sketch_estimator_config.sketch_factory(sketch_random_seed)
       sketch.add_ids(campaign_ids)
       sketches.append(sketch)
 
-    # Optionally noise the sketches if it exists in the estimator_config.
-    if hasattr(self.estimator_config, 'sketch_noiser') and self.estimator_config.sketch_noiser:
+    # Optionally noise the sketches.
+    if hasattr(self.estimator_config,
```

Comment:
self.sketch_estimator_config?

Reply:
sorry I'm not sure what you're suggesting: can you clarify?

Reply:
Did you actually mean self.sketch_estimator_config rather than  self.estimator_config here? I saw self.estimator_config was never set in any method and this access would probably throw AttributeError?

Reply:
there was a merge error I should have caught, the new commit should address this.  Thanks!